x-pack/plugins/security_solution/public/common/components/header_section/index.tsx

```diff
@@ -97,7 +100,7 @@ const HeaderSectionComponent: React.FC<HeaderSectionProps> = ({
             )}
           </EuiFlexItem>
 
-          {id && (
+          {id && showInspectButton && (
```

Comment:
Can we test this with the unit test?

Reply:
https://github.com/elastic/kibana/pull/126507/commits/96ed710f78494e491557b97ffecd200a833c0aaf